packages/rocketchat-crowd/server/crowd.js

```diff
@@ -75,15 +76,68 @@ export class CROWD {
 			return;
 		}
 
-		logger.info('Going to crowd:', username);
-		const auth = this.crowdClient.user.authenticateSync(username, password);
+		logger.info('Extracting crowd_username');
+		let user = null;
+		let crowd_username = username;
+		try {
+			user = Meteor.users.findOne({
+				username,
+			});
+			crowd_username = user.crowd_username;
+		} catch (error) {
+			logger.debug(error);
+			logger.debug('Could not find user by username. Checking email');
+		}
+
+		if (!crowd_username) {
```

Comment:
This check should be moved below since the check by email could find an existent user without the property `crowd_username`